### DISCUSSION TITLE
Python ModuleNotFoundError 'distutils' on MacOS M2

### DISCUSSION THREAD

**Discussion by DavidGiraldoCode** [inviwo/inviwo#1481] [Q&A]
I tried to build using Cmake in the terminal:

```bash
cmake -S /path/Inviwo -B /path/Inviwo-output -G Xcode -DCMAKE_C_COMPILER=$(xcrun -find cc) -DCMAKE_CXX_COMPILER=$(xcrun -find c++)
```
**Context:**
- Qt: stable 6.6.1
- Cmake: stable 3.28.1
- Xcode 14.3.1
- Build version 14E300c

But it keep throwing the following error:
```bash
-- pybind11 v2.6.1
CMake Error at ext/pybind11/tools/pybind11NewTools.cmake:106 (message):
  pybind11 could not query the module file extension, likely the
  'distutils'package is not installed.  Full error message:

  Traceback (most recent call last):

    File "<string>", line 1, in <module>

  ModuleNotFoundError: No module named 'distutils'

Call Stack (most recent call first):
  ext/pybind11/tools/pybind11Common.cmake:196 (include)
  ext/pybind11/CMakeLists.txt:169 (include)
```

I had Python 3.12 installed, but noticed that 'distutils' was deprecated and has been removed.

Then, I installed Python 3.11, but the error persisted.

**Comment by petersteneteg** [comment]
Hi, thanks for reporting. We probably need to update the pybind11 submodule

It should work I you use vcpkg to acquire the dependencies. 
have a look at https://inviwo.org/manual-gettingstarted-build.html#building-1

**Comment by petersteneteg** [comment]
fixed in #1484
